CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#19](https://github.com/rubocop-hq/rubocop-minitest/pull/19): Fixes typo `collection.includes?` to `collection.include?` in `Minitest/AssertIncludes` cop. ([@abhaynikam][])
```

Comment:
It would be better to write a false negative correction rather than a typo :-) The following is an example.

```
Fix a false negative for `Minitest/AssertIncludes` when using `include` method in arguments of `assert` method.
```

Reply:
Thanks, @koic 🎉 . Updated the changelog